operatorhub/openshift/config.yaml

```diff
@@ -246,7 +246,13 @@ image-substitutions:
         containerName: openshift-pipelines-operator-lifecycle
         envKeys:
           - IMAGE_RESULTS_API
-
+- image: registry.redhat.io/openshift-pipelines/pipelines-results-retention-policy-agent-rhel8@
+  replaceLocations:
+    envTargets:
+      - deploymentName: openshift-pipelines-operator
+        containerName: openshift-pipelines-operator-lifecycle
+        envKeys:
+          - IMAGE_RETENTION_POLICY_AGENT
```

Comment:
@pratap0007 please add the details in https://github.com/tektoncd/operator/blob/main/docs/AirGapImageConfiguration.md
also please provide detailed description about this image, something like, why do we need this?